src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -276,4 +281,15 @@ private int vacationsDayPerSeniority (LocalDate startDate, LocalDate currentDate
         int yearDiff = startDate.until(currentDate).getYears();
         return yearDiff >= 5 ? 20 : (yearDiff >= 2 ? 15 : activeContract.get().getSeniority().getVacationDays());
     }
-}
+
+    public List<Employee> getUnassignedEmployees(){
+        List<Employee> fullEmployeesList = employeeRepository.findAllByDeletedIsFalseAndActiveIsTrue();
```

Comment:
should we create a new repository method in order to replace all the operations in this method? @agustinb4